### ISSUE TITLE
Deep clean before furniture goes in

### ISSUE THREAD

**Issue by iteles** [dwyl/home#104]
The dust has now settled for a couple of weeks and the next steps are to carry out a deep clean.

The only thing left to do that will create dust on the inside are the staircases #30, but we'll need more than one deep clean anyway. 

Therefore ideally we'd like to get one big first clean done before the end of the year so that we can start moving in the furniture.

**Comment by iteles** [comment]
I have called:
+ [Confort Limpezas](https://empresa-limpezas.pt/limpezas-pos-obras-braga/) who are no longer in business
+ [Matrix Clean](https://matrixclean.pt/) I sent a form with a quotation request but they never got back to me
+ [MarClean](https://marclean.pt/limpezas-pos-obra/), who asked us to send them an email with the details for a quote
+ [HouseShine](https://houseshine.pt/franquias/braga/), who we're meeting tomorrow at the site at 10h30 to get a quote

**Comment by iteles** [comment]
Budgets are in from HouseShine:
![image](https://user-images.githubusercontent.com/4185328/70476419-8d093f00-1ace-11ea-9d6d-b628c82ff275.png)

We need to actually be there to both let them in and sign off at the end.

**Comment by nelsonic** [comment]
@iteles it's quite a big house with lots of construction dust to clean so the price seems _fair_. 
While I think I _could_ clean it all in 44h, it _probably_ wouldn't be a good use of my/our time at that price. 💭  
I would _definitely_ like to be there (_to clean out the media room_) and setup my standing desk.

**Comment by iteles** [comment]
@nelsonic I was wondering if it would be worth this happening before the end of the year of if we should book it for the week of the 6th January so things are fresh for the new year!

**Comment by nelsonic** [comment]
@iteles I think January is best in terms of our timelines,
but you could ask them if they have availability for _this_ week.

**Comment by iteles** [comment]
They did have availability this week, hence my question.

**Comment by nelsonic** [comment]
Ah, yes, book it for the week beginning 6<sup>th</sup> January. Preferably 7<sup>th</sup> so we don't have to be there at 09:00 on the Monday morning ...

**Comment by iteles** [comment]
Booked for the 7th, starting after lunch.

**Comment by nelsonic** [comment]
Cool. I will be there.

**Comment by iteles** [comment]
As per updated calendar invites yesterday morning, this was moved to Wednesday morning, 8-Jan-2020, 09h00 by the cleaning company.

**Comment by nelsonic** [comment]
That's _not_ what the calendar invite I received _yesterday_ said. 
![Screenshot_20200107-070800~2](https://user-images.githubusercontent.com/194400/71875643-b2648980-311c-11ea-92b7-054b749c8975.png)

**Comment by iteles** [comment]
Immediately after sending out this invite, I saw in my calendar that the date was incorrect and updated it again, literally within one minute. So strange that you received one and not the other 😞 Sorry about that.

**Comment by nelsonic** [comment]
Sounds like the combination of Google Calendar and Email is letting us down pretty badly ... 😢 
Oh well. _More_ wasted time due to poor communication (tools). Tomorrow it is then.

**Comment by iteles** [comment]
First pass of this has now been completed 👍🏻
Will open a new issue for the next clean after #30 is complete.
